Sources/Commands/SwiftTestCommand.swift

```diff
@@ -1342,10 +1351,14 @@ private extension Basics.Diagnostic {
 /// - Returns: The paths to the build test products.
 private func buildTestsIfNeeded(
     swiftCommandState: SwiftCommandState,
-    buildParameters: BuildParameters,
+    productsBuildParameters: BuildParameters,
+    toolsBuildParameters: BuildParameters,
     testProduct: String?
 ) throws -> [BuiltTestProduct] {
-    let buildSystem = try swiftCommandState.createBuildSystem(productsBuildParameters: buildParameters)
+    let buildSystem = try swiftCommandState.createBuildSystem(
+      productsBuildParameters: productsBuildParameters,
```

Comment:
2-space formatting is inconsistent here w/ the rest of the code base

Reply:
Oops. Will fix.